scalyr_agent/scalyr_client.py

```diff
@@ -1540,40 +1542,54 @@ def __set_attributes(self, thread_id, attributes, log_line_attributes=False):
 
         self.__serialization_base = tmp_buffer.getvalue()
 
-    def add_missing_attributes(self, attributes, log_line_attributes=False):
-        """ Adds items attributes to the base_event's attributes if the base_event doesn't
+    def __get_attributes_to_serialize(self):
```

Comment:
Add comment for this method.. something like `Return the attributes that should be included in the serialization for this specific event.  This does not include attributes from its parent if using the logfile_addevents_format.`